src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayPay = 40;
+  const littleSale = 20;
+  const bigSale = 50;
+  const totalPrice = days * oneDayPay;
+
+  if (days < 3) {
+    return totalPrice;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalPrice - littleSale;
+  }
+
+  if (days >= 7) {
+    return totalPrice - bigSale;
+  }
```

Comment:
You should have at least one consistent return statement (outside of if) and you've already covered all the cases, so this check is redundant
```suggestion
    return totalPrice - bigSale;
```